From 55dffb6532f0f3f7b6500e977b71faed6acac024 Mon Sep 17 00:00:00 2001
From: Andy Grover <agrover@redhat.com>
Date: Tue, 2 Jul 2013 14:12:07 -0700
Subject: [PATCH] check-if-tpglun-exists

---
 targetcli/ui_target.py | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/targetcli/ui_target.py b/targetcli/ui_target.py
index 1ffc0af..9da4868 100644
--- a/targetcli/ui_target.py
+++ b/targetcli/ui_target.py
@@ -790,14 +790,20 @@ class UILUNs(UINode):
                                    self.shell.prefs['auto_add_mapped_luns'])
 
         try:
-            storage_object = self.get_node(storage_object).rtsnode
+            so = self.get_node(storage_object).rtsnode
         except ValueError:
             self.shell.log.error("Invalid storage object %s." % storage_object)
             return
 
+        sos = (l.storage_object for l in self.parent.rtsnode.luns)
+        so_paths = ("/backstores/%s/%s" % (x.plugin, x.name) for x in sos)
+        if storage_object in so_paths:
+            raise ExecutionError("lun for storage object %s already exists" \
+                                     % storage_object)
+
         if lun and lun.lower().startswith('lun'):
             lun = lun[3:]
-        lun_object = LUN(self.tpg, lun, storage_object)
+        lun_object = LUN(self.tpg, lun, so)
         self.shell.log.info("Created LUN %s." % lun_object.lun)
         ui_lun = UILUN(lun_object, self)
 
